### PR TITLE
Move style tags to head

### PR DESCRIPTION
**Pull request by jbothma**
## Description

Style tags are moved to the head to be valid html and this is a blocker for moving to parcel2.

Nothing should change from the user's perspective.

## How is it tested automatically?

> Is the particular functionality/structure of the backend API this relies on tested?
> What unit, integration, GUI, end-to-end tests were added to test the functionality implemented/fixed in this PR?
> What other tests does it rely on so that the new tests are sufficient?

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally
- [ ] 👩‍🎨 does the design match the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out
- [ ] commit messages are meaningful

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

**Review by michaelglenister — APPROVED** [review]
Looks good, it would be nice to have the CSS separated into stylesheets at some point

File: src/index.html [hunk 1]
```diff
@@ -22,7 +22,342 @@
   <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
   <link href="images/favicon.png" rel="shortcut icon" type="image/x-icon">
   <link href="images/webclip.png" rel="apple-touch-icon">
-  <script src="https://kit.fontawesome.com/95e41092af.js" crossorigin="anonymous"></script>
+  <script defer="" src="https://kit.fontawesome.com/95e41092af.js" crossorigin="anonymous"></script>
```
[the remark this answers is not in the document]
Comment (jbothma) [reply src/index.html:25]: Looks like some post-processing from webflow is adding it. doesn't seem like it's harmful.
Screenshot_2022-10-04_11-23-14

**Comment by emre2038** [comment]
@jbothma could you resolve the conflicts please

**Comment by jbothma** [comment]
@emre2038 resolved conflicts

**Comment by jbothma** [comment]
@emre2038 done resolving conflicts

**Comment by jbothma** [comment]
I went with my script tag replace + add approach rather than your update approach because webflow shouldn't even be including that gcro script but I just haven't dared removing it yet

**Pull request merged**